Doc/whatsnew/3.12.rst

```diff
@@ -385,6 +385,15 @@ typing
   same name on a base class, as per :pep:`698`. (Contributed by Steven Troxler in
   :gh:`101564`.)
 
+unittest
+--------
+
+:meth:`~unittest.TestResult.addError`,
+:meth:`~unittest.TestResult.addFailure` and
+:meth:`~unittest.TestResult.addSubTest` now accept either an exception
+instance or a ``(typ, val, tb)`` tuple. (Contributed by Irit Katriel in
```

Comment:
Make clear what is new while being clear that still only 1 is passed.
```suggestion
:meth:`~unittest.TestResult.addSubTest` now accept an exception instead of
a ``(typ, val, tb)`` tuple. (Contributed by Irit Katriel in
```

Reply:
I think "instead of" may sound like the old format is no longer accepted. But agree I would emphasise what is new and what is old.